src/app/client/src/app/modules/cbse-program/components/question-preview/question-preview.component.ts

```diff
@@ -77,11 +81,22 @@ export class QuestionPreviewComponent implements OnInit, OnChanges {
           this.theme = theme;
           const context = this.getContext();
           this.playerConfig =  this.setPlayerConfig(context, theme);
+          this.assessDetails = this.prepareAssessDetails();
         })
       }
     }
   }
 
+  prepareAssessDetails(){
+    const assessDetails = {}
+    if(this.questionMetaData){
+      assessDetails['correct_response'] = this.questionMetaData.data ? parseInt(this.questionMetaData.data.responseDeclaration.responseValue.correct_response.value) + 1 : parseInt(this.selectedAttributes.previewQuestionData.result.assessment_item.responseDeclaration.responseValue.correct_response.value) + 1;
+      assessDetails['learningOutcome'] = (this.questionMetaData.data && this.questionMetaData.data.learningOutcome) ? this.questionMetaData.data.learningOutcome[0] : undefined;
+      assessDetails['bloomsLevel'] = this.questionMetaData.data ? this.questionMetaData.data.bloomsLevel[0] : this.selectedAttributes.previewQuestionData.result.assessment_item.bloomsLevel[0]; 
```

Comment:
this.selectedAttributes.previewQuestionData.result.assessment_item is being accessed twice. instead of writing completely twice, assign it to some variable and use it.